examples/dev/data-prepper-emf-monitoring/docker-compose.yml

```diff
@@ -10,8 +10,8 @@ services:
     environment:
       AWS_EMF_AGENT_ENDPOINT: "tcp://fluent-bit:25888"
     volumes:
-      - ./pipelines-raw-trace-stdout.yaml:/usr/share/data-prepper/pipelines.yaml
-      - ./data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml
+      - ./pipelines-raw-trace-stdout.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml
```

Comment:
We should probably start to update our samples to do either of the following:
1. Create a directory-based volume. But, this might always work since we'd have to ensure only pipelines are there.
2. Use the same name in the Docker image as is used in the host. I recommend this so that we move away from the old approach.

So, if we go with suggestion 2 (which is probably easiest), we'd get:

```
./pipelines-raw-trace-stdout.yaml:/usr/share/data-prepper/pipelines/pipelines-raw-trace-stdout.yaml
```

Reply:
Good point. Addressed this and all the comments below in the latest commit.